validate.py

```diff
@@ -2,20 +2,41 @@
 from dateutil import parser
 from flask import Flask, request, jsonify
 import settings
-import json
 import logging
 import logging.handlers
 from logging import Formatter
 
 app = Flask(__name__)
 
+KNOWN_SURVEYS = ['023']
+KNOWN_INSTRUMENTS = ['0203', '0213', '0205', '0215', '0102', '0112']
+
 
 # Parses a timestamp, throwing a value error
 # if unrecognised
 def Timestamp(value):
     return parser.parse(value)
 
 
+def ValidSurveyId(value):
+    if value not in KNOWN_SURVEYS:
+        raise ValueError('Invalid survey id')
+
+
+def ValidInstrumentId(value):
+    if value not in KNOWN_INSTRUMENTS:
+        raise ValueError('Invalid instrument id')
+
+
+def ValidData(data):
```

Comment:
I think `ValidData` sounds too generic, as this is validating that the survey data is valid what about `ValidSurveyData`?